db/change_cache.go

```diff
@@ -300,23 +294,26 @@ func (c *changeCache) InsertPendingEntries(ctx context.Context) error {
 
 // Cleanup function, invoked periodically.
 // Removes skipped entries from skippedSeqs that have been waiting longer
-// than MaxChannelLogMissingWaitTime from the queue.  Attempts view retrieval
-// prior to removal.  Only locks skipped sequence queue to build the initial set (GetSkippedSequencesOlderThanMaxWait)
-// and subsequent removal (RemoveSkipped).
+// than MaxChannelLogMissingWaitTime from the queue.
 func (c *changeCache) CleanSkippedSequenceQueue(ctx context.Context) error {
 
-	oldSkippedSequences := c.GetSkippedSequencesOlderThanMaxWait()
-	if len(oldSkippedSequences) == 0 {
+	base.InfofCtx(ctx, base.KeyCache, "Starting CleanSkippedSequenceQueue for database %s", base.MD(c.db.Name))
+
+	maxWait := int64(c.options.CacheSkippedSeqMaxWait.Seconds())
+	if maxWait < 1 {
+		maxWait = 1
+	}
+	numOldSkippedSequences := c.skippedSeqs.SkippedSequenceCompact(ctx, maxWait)
```

Comment:
```suggestion
	compactedSequences := c.skippedSeqs.SkippedSequenceCompact(ctx, maxWait)
```
I think numCompactedSequences or maybe numEvictedSequences would be a clearer name here.